common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/WorksQuery.scala

```diff
@@ -0,0 +1,57 @@
+package uk.ac.wellcome.elasticsearch
```

Comment:
Has this been removed in the API project? Might be being stupid but can't see it